packages/schema/lib/utils/makeValidator.js

```diff
@@ -130,9 +130,14 @@ const makeValidator = (mainSchema, subSchemas) => {
   return {
     validate: (definition) => {
       const results = v.validate(definition, mainSchema);
-      const allErrors = results.errors.concat(
+      let allErrors = results.errors.concat(
         functionalConstraints.run(definition, mainSchema)
       );
+
+      allErrors = allErrors.filter((error) => {
+        // remove errors with message 'requires property "perform"' when definition[action] has bulk/performBulk set.
```

Comment:
@eliangcs As we still need to make the property `perform` required, I was thinking of removing the errors with the message 'requires property "perform"' when definition[action] has bulk/performBulk set.

Reply:
Good idea, thanks.

Reply:
Done